cookbooks/ntp/recipes/default.rb

```diff
@@ -52,6 +52,10 @@
   subscribes :restart, 'template[/etc/ntp.conf]'
 end
 
+service "ntp" do
+    action :start
```

Comment:
@DalianisDim why don't we just enable the service?

Reply:
Submitted improvements to cover all cases. Service is enabled and started right away